<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatureSelection</class>
 <widget class="QWidget" name="FeatureSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主界面</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>本工具用于分析航空公司客户特征并预测客户流失情况</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_home">
         <property name="text">
          <string>主界面</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_Kmeans">
         <property name="text">
          <string>聚类分析</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_predict">
         <property name="text">
          <string>客户流失预测</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,5,0,0,0,0">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>步骤一导入数据：可选导入原始数据（如temp/data_init.xls）并进行清洗或选择直接导入清洗后的数据（如temp/data_cleaned.xls）</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="PB_openfile">
           <property name="text">
            <string>导入数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_clean">
           <property name="text">
            <string>数据清理</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="DataTableWidget" name="pandasTableWidget" native="true"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>步骤二特征选择：可选输入五组特征的列名，用空格分割或直接采用预置的LRFMC模型进行特征选择</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="TE_featureSelection">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="PB_Confirm">
           <property name="text">
            <string>确认特征选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_LRFMC">
           <property name="text">
            <string>LRFMC特征选择</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="LB_stat">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>STATE</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataTableWidget</class>
   <extends>QWidget</extends>
   <header>qtpandas.views.DataTableView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
